zenoh-flow/src/runtime/dataflow/instance/runners/sink.rs

```diff
@@ -102,7 +102,7 @@ impl Runner for SinkRunner {
         HashMap::with_capacity(0)
     }
 
-    async fn get_input_links(&self) -> HashMap<PortId, LinkReceiver<Message>> {
+    async fn take_input_links(&self) -> HashMap<PortId, LinkReceiver<Message>> {
```

Comment:
As discussed, we should transform `self.link` and make it a `Mutex<Option<…>` to actually "take" it